### PR TITLE
README update

### PR DESCRIPTION
**Pull request by lalli-flores**
This also includes moving the release process section into the Contribution guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,46 +36,68 @@ aimed at react developers that want to embed the widgets into their applications
...
+has be installed as a peer dependency.
+
 ## Webex Components vs Webex Widgets
 
-[Webex Components](https://github.com/webex/components#webex-components) is a set of React components that follow webex standard styling without any data being injected to them.
-[Webex Widgets](https://github.com/webex/widgets#webex-widgets) are based on [Webex components](https://github.com/webex/components#webex-components) but include the adapter that talks to our [Javascript SDK](https://github.com/webex/webex-js-sdk).
-This means that the Webex Widgets use the [SDK Component Adapter](https://github.com/webex/sdk-component-adapter#webex-sdk-component-adapter) to inject the [Webex data](https://github.com/webex/webex-js-sdk#webex-js-sdk) for you.
+In addition to the Webex Widgets, we also offer the
+[Webex Component System](https://github.com/webex/components#webex-components).
+The Webex Component System (sometimes shorten as _Webex Components_) is a set of
```
Comment (adamweeks) [inline README.md]:
```suggestion
The Webex Component System (sometimes shortened as _Webex Components_) is a set of
```

File: README.md [hunk 2]
```diff
@@ -34,46 +36,68 @@ aimed at react developers that want to embed the widgets into their applications
...
-In order to properly style Webex Widgets, we need to import all the fonts, icons, images and core CSS manually.
+In order to properly style Webex Widgets, you will need to import our CSS separately.
 Import `@webex/widgets/dist/webexWidgets.css` into your main entry file.
 
 There are two ways to do this:
 
 #### JavaScript
 
-In your `App.js`, add the following import:
+In your `App.js(x)`, add the following import:
```
Comment (adamweeks) [inline README.md]: What is this addition?

Comment (lalli-flores) [reply README.md]: This is supposed to be `App.js` or `App.jsx`, if it doesn't make sense I can revert it

**Pull request merged**